libraries/nomad_job.rb

```diff
@@ -20,16 +21,40 @@ def initialize(url, job_name)
   end
 
   def running?
-    job = http_json("#{@url}/v1/job/#{@job_name}")
+    job = get_job
     job['Status'] == 'running'
   end
 
+  def job_status
+    job = get_job
+    print_allocation_logs(job['ID'])
+    job_status = job['Status']
```

Comment:
```suggestion
    job['Status']
```

This will return the content of `job['Status']` (Ruby :man_shrugging: )
See: http://ruby-for-beginners.rubymonstas.org/writing_methods/return_values.html